src/citrine/informatics/data_sources.py

```diff
@@ -53,22 +53,30 @@ class CSVDataSource(Serializable['CSVDataSource'], DataSource):
         link to the CSV file to read the data from
     column_definitions: Mapping[str, Descriptor]
         Map the column headers to the descriptors that will be used to interpret the cell contents
+    identifiers: Optional[List[str]]
+        List of one or more column headers whose values uniquely identify a row. These may overlap
+        with ``column_definitions`` if a column should be used as data and as an identifier,
+        but this is not necessary. Identifiers should be globally unique. No two rows should
```

Comment:
Is "globally unique" literal here, or does it just mean unique within the {CSV,dataset,etc}?

Reply:
Also, same comment about "should" as above. I suggest a stronger word like "must", or explain when duplication might be reasonable.

Reply:
It must be unique within the context of the training data. I'll clarify that in the docs.